modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -191,6 +189,22 @@ ContentDashboardAdminManagementToolbarDisplayContext contentDashboardAdminManage
 									dropdownItems="<%= contentDashboardAdminDisplayContext.getDropdownItems(contentDashboardItem) %>"
 									propsTransformer="js/transformers/ActionsComponentPropsTransformer"
 								/>
+
+								<div>
+
+									<%
+									Map<String, Object> quickActionsProps = HashMapBuilder.<String, Object>put(
+										"items", contentDashboardAdminDisplayContext.getDropdownItems(contentDashboardItem)
+									).put(
+										"namespace", liferayPortletResponse.getNamespace()
```

Comment:
Isn't `namespace` already passed down to the component?

Reply:
You're right, is `portletNamespace` is being passed, thx!